website/highlights/2020-06-10-consolidate-and-beautify-validate.md

```diff
@@ -0,0 +1,36 @@
+---
+last_modified_on: "2020-07-13"
+$schema: "/.meta/.schemas/highlights.json"
+title: "Beautification of the validate command"
+description: "A little polish on a useful feature."
+author_github: "https://github.com/hoverbear"
+hide_on_release_notes: false
+pr_numbers: [2622]
+release: "0.10.0"
+tags: ["type: enhancement", "domain: ux"]
+---
+
+We gave `vector validate` some touching up to make it look better, and feel nicer to use.
```

Comment:
Noting, this was heavily inspired by `linkerd validate`, so might be worth a shoutout to them.